docs/help-center/tim-features/use-hba.mdx

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_label: Use HBA
+title: Use HBA
+hide_title: true
+displayed_sidebar: helpcenterSidebar
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Use HBA</h2>
+  </div>
+</div>
+
+## Register HBA card
+
+To connect to VZD Fhir Directory you have to add your HBA. This needs to be done with
+a mobile device since you need NFC connection to read the card.  
+After registering, you will be able to find users and contact points of other organizations.
```

Comment:
I have been told that this is not fully correct. You can find people and contact points always, however, you cannot contact other eHBA users, unless you also connect your own. The contact points can also be contacted by people without eHBA, these are public within TIM.